common/src/main/java/io/seata/common/loader/EnhancedServiceLoader.java

```diff
@@ -193,8 +194,9 @@
         return loadFile(service, activateName, loader, null, null);
     }
 
-        @SuppressWarnings("rawtypes")
-    private static <S> S loadFile(Class<S> service, String activateName, ClassLoader loader, Class[] argTypes, Object[] args) {
+    @SuppressWarnings("rawtypes")
+    private static <S> S loadFile(Class<S> service, String activateName, ClassLoader loader, Class[] argTypes,
```

Comment:
Why change the line here ？
I think we should have more characters on each line to avoid more line breaks.